R/tm_a_regression.R

```diff
@@ -539,6 +534,8 @@ srv_a_regression <- function(id,
           }
         })) %>%
         teal.code::eval_code(quote(summary(fit)))
```

Comment:
I don't think we had an output for `lm` model in the previous version of this module, but this line is in the output code, so maybe we would like to deliver results of this model


```suggestion
        teal.code::eval_code(quote(summary(fit)), keep_output = TRUE)
```

<img width="496" alt="image" src="https://github.com/user-attachments/assets/4432d3dc-992d-44ed-aff2-44412a093d7e" />


this `lm` summary is also a part of the module

<img width="677" alt="image" src="https://github.com/user-attachments/assets/c858842f-9eec-421c-b211-f893399b1a1e" />

Reply:
Yes, we should add everything which is

Reply:
ok, so I think we will need to add `toHTML` method for `summary.lm` class in `teal.reporter`

```r
#' @method .toHTML summary.lm
#' @keywords internal
.toHTML.summary.lm <- function(x, ...) {
  shiny::tags$pre(x)
}
```

Reply:
it actually needs to me something closer to 

```r
shiny::tags$pre(paste(capture.output(print(x)), collapse = "\n"))
```

Reply:
Added a method for summary.lm in teal.reporter https://github.com/insightsengineering/teal.reporter/pull/335/commits/224a74456c9aed47534cf809c8cb0a38dfa470a4

and now the output looks like this



![image](https://github.com/user-attachments/assets/61484204-7ccb-441a-86f1-982208af1446)